CONTRIBUTING.md

```diff
@@ -85,13 +85,20 @@ Docker is _not_ required to work on this gem.
 
 The `docker-compose` approach should link to enough of your networking configuration that you can release the gem.
 However, you have to do some of the configuration yourself, because it's dependent on your host operating system.
+
```

Comment:
I feel the Docker documentation has grown so large it should be extracted into its own document (DOCKER.md) and then be referred from the CONTRIBUTING.md document.

Reply:
Good idea. I'll do that in a separate PR.